app/migrations/versions/2460fe231511_.py

```diff
@@ -0,0 +1,94 @@
+"""empty message
+
+Revision ID: 2460fe231511
+Revises: 18c8798add0d
+Create Date: 2019-11-04 18:02:45.983420
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '2460fe231511'
+down_revision = '18c8798add0d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
thought this PR was for tests? why is exchange stuff also in it?

Reply:
in the future, for bigger PRs, can we add a description of what are the major changes in the PR? easier for people to get context for reviewing

Reply:
yeah will do. hopefully there won't be many big prs like this in the future. also this migration file i think just didn't get commited properly last time